packages/core/core-flows/src/cart/utils/prepare-confirm-inventory-input.ts

```diff
@@ -67,6 +80,22 @@ export const prepareConfirmInventoryInput = (data: {
         hasSalesChannelStockLocation = true
       }
 
+      if (location_levels) {
+        const availabilty = MathBN.sub(
+          location_levels.raw_stocked_quantity ??
+            location_levels.stocked_quantity ??
+            0,
+          location_levels.raw_reserved_quantity ??
+            location_levels.reserved_quantity ??
+            0
+        )
+
+        mapLocationAvailability.set(
```

Comment:
q: if the variant has an inventory kit i.e. multiple inventory items, wouldn't this set only set availability for that location for the last item only?